FIDO2_authenticate_credential.md

```diff
@@ -0,0 +1,26 @@
+## FIDO2 Authenticate Credential
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
Agree.